packages/fxa-content-server/webpack.config.js

```diff
@@ -52,6 +52,8 @@ const webpackConfig = {
       'node_modules',
     ],
     alias: {
+      '@fxa-components': path.resolve(__dirname, '..', 'fxa-components'),
+      '@fxa-shared': path.resolve(__dirname, '..', 'fxa-shared'),
```

Comment:
These two are unrelated to the PR, but opens up the ability to import from these external packages. TSConfig is also reflected.